### DISCUSSION TITLE
Self upload unidentified file

### DISCUSSION THREAD

**Discussion by OveTomash84** [desyncr/mobile-node#108] [Q&A] [answered]
Hello. Today i found that FN-mobile try to upload some file. I did not put any file for upload. It was in broken uploads “Chat-0“ 121B. What was that?

**Comment by desyncr** [comment, marked as answer]
By the file name it seems it's related to https://github.com/DennisRein/free-chat-2. Freechat uses these uploads to transfer tokens as invites.
